### PR TITLE
Derived market query table

### PR DESCRIPTION
**Pull request by nuevoalex**

### CODE REVIEW

File: packages/augur-sdk/src/state/db/MarketDB.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    // TODO Register callback after CurrentOrders is updated during blockstream. Use augurEmitter to subscribe to it and do post processing on the orderbook.
+
+    this.flexSearch = new FlexSearch({
+      doc: {
+        id: "id",
+        start: "start",
+        end: "end",
+        field: [
+          "market",
+          "topic",
```
Comment (bthaile) [inline packages/augur-sdk/src/state/db/MarketDB.ts:33]: just a node: We'll need to refactor when topic is removed and 2-3 categories is added via extraInfo

File: packages/augur-sdk/src/state/db/MarketDB.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          "description",
+          "longDescription",
+          "resolutionSource",
+          "_scalarDenomination",
+          "tags"
```
Comment (bthaile) [inline packages/augur-sdk/src/state/db/MarketDB.ts:38]: note: tags are getting removed as well.

File: packages/augur-sdk/src/state/db/MarketDB.ts [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  constructor(db: DB, networkId: number) {
+    super(db, networkId, "Markets", ["MarketCreated", "MarketVolumeChanged", "MarketOIChanged"], ["market"]);
+    // TODO Register callback after CurrentOrders is updated during blockstream. Use augurEmitter to subscribe to it and do post processing on the orderbook.
+
+    this.flexSearch = new FlexSearch({
+      doc: {
+        id: "id",
+        start: "start",
+        end: "end",
+        field: [
```
Comment (bthaile) [inline packages/augur-sdk/src/state/db/MarketDB.ts:31]: Will category outcomes be in flex search?

**Review by justinbarry — APPROVED** [review]
LGTM

**Pull request merged**